themes/default/content/resources/getting-started-with-infrastructure-as-code-on-aws-7-19-22/index.md

```diff
@@ -0,0 +1,80 @@
+---
+# Name of the webinar.
+title: "Getting Started with Infrastructure as Code on AWS"
+meta_desc: "In this workshop, you will learn the fundamentals of Infrastructure as Code through a series of guided exercises using Pulumi’s Cloud Engineering platform. You will be introduced to Pulumi, an infrastructure as code platform, where you can use familiar programming languages to provision modern cloud infrastructure."
```

Comment:
```suggestion
meta_desc: "In this workshop, you will learn the fundamentals of Infrastructure as Code through a series of guided exercises using the Pulumi Cloud Engineering platform."
```

Reply:
I agree, I think that first sentence is enough and actually like it better since it is more concise. Thanks @cnunciato.